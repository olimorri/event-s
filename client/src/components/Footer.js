import {
  Box,
  Container,
  Stack,
  Text
} from '@chakra-ui/react';



export default function SmallWithSocial() {
  return (
    <Box
    bg={'custom.100'}
    justify={'center'}
    >
      <Container
        as={Stack}
        maxW={'6xl'}
        py={4}
        direction={{ base: 'column', md: 'row' }}
        spacing={4}
        justify={'center'}
        align={{ base: 'center', md: 'center' }}>
        <Text >Andras</Text>
        <Stack direction={'row'} spacing={6}>
        </Stack>
      </Container>
    </Box>
  );
}