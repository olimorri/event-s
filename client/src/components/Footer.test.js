import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Footer from './Footer';

const FooterTest = () => (
  <BrowserRouter>
    <Footer />
  </BrowserRouter>
);

describe('Snapshot', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<FooterTest />);
    expect(asFragment(<FooterTest />)).toMatchSnapshot();
  });
});

describe('Unit Testing', () => {
  it('should render the footer text', () => {
    render(<FooterTest />);
    const footerText = screen.getByText(/andras/i);
    expect(footerText).toBeInTheDocument();
  });
});
