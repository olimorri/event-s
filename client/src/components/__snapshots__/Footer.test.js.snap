// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-1apad05"
    justify="center"
  >
    <div
      class="chakra-stack chakra-container css-1xqezda"
    >
      <p
        class="chakra-text css-0"
      >
        Andras
      </p>
      <div
        class="chakra-stack css-57svvl"
      />
    </div>
  </div>
</DocumentFragment>
`;
